satellite/audit/disqualification_test.go

```diff
@@ -65,10 +62,10 @@ func TestDisqualificationTooManyFailedAudits(t *testing.T) {
 		prevReputation := calcReputation(dossier)
 
 		// Report the audit failure until the node gets disqualified due to many
-		// failed audits
+		// failed audits.
 		iterations := 1
 		for ; ; iterations++ {
-			_, err := satellitePeer.Audit.Service.Reporter.RecordAudits(ctx, report)
+			_, err := satellitePeer.Audit.Worker.Reporter.RecordAudits(ctx, report)
```

Comment:
If we make `Reporter` exported on `peer.Audit`, and private on `Worker`, we can just use `satellitePeer.Audit.Reporter.RecordAudits` here

Reply:
ok cool!